### ISSUE TITLE
Comparisons between Decimals get translated to ".CompareTo"

### ISSUE THREAD

**Issue by jptarqu** [fable-compiler/Fable#747]
### Description

When a line in F# compares a decimal to another decimal, it gets transpiled to a literal JavaScript number with a ".CompareTo" call.

### Repro code
F#

```
let amtReq = 5M
let min = 0M
let max = 25000M
let dummy = if ((amtReq) >= (min ) && (amtReq ) < (max )) then true else false
```

### Expected and actual results

Gets translated to JS as:
```
var amtReq = 5;
var min = 0;
var max = 25000;
var dummy = (amtReq.CompareTo(min) >= 0 ? amtReq.CompareTo(max) < 0 : false) ? true : false;
```

Which causes the following error: " amtReq.CompareTo is not a function "

I was expecting:
```
var amtReq = 5;
var min = 0;
var max = 25000;
var dummy = (amtReq >= min  ? amtReq < max  : false) ? true : false;
```

### Related information 

Also, if I add "+ 0M" to the elements being compared, then it gets translated correctly:

F#
```
let amtReq = 5M
let min = 0M
let max = 25000M
let dummy = if ((amtReq + 0M) >= (min + 0M) && (amtReq + 0M) < (max + 0M)) then true else false
```

JS:
```
var amtReq = 5;
var min = 0;
var max = 25000;
var dummy = (amtReq + 0 >= min + 0 ? amtReq + 0 < max + 0 : false) ? true : false;
```

* fable-compiler version: fable-compiler 0.7.49
* fable-core version: 0.7.28
* Operating system: Windows 10

**Comment by alfonsogarciacaro** [comment]
Sorry for the late reply, I've fixed this in Fable 0.7.50. Can you please give a try?

**Comment by jptarqu** [comment]
It works! Thank you!
